src/components/attendee/Links/ProfileModalEditLinks.tsx

```diff
@@ -2,7 +2,7 @@ import React, { useEffect, useMemo } from "react";
 import { FieldErrors, UseFormRegister } from "react-hook-form";
 import classNames from "classnames";
 import { Button } from "components/attendee/Button/Button";
-import { ProfileModalEditLink } from "components/attendee/NavOverlay/ProfileOverlay/Links/ProfileModalEditLink";
+import { ProfileModalEditLink } from "components/attendee/Links/ProfileModalEditLink";
```

Comment:
[Q] is `<Links />` a component? If it is simply a grouping directory, might be easier to not confuse if named `/links/`